recipe/meta.yaml

```diff
@@ -27,37 +27,41 @@ requirements:
     - pip
     - wheel
     - numpy 1.16
+    - cython
```

Comment:
I do not understand why it was working before.

Reply:
Before we did not need cython for builds.